frontend/src/components/TransactionLinks/TransactionLink.vue

```diff
@@ -115,6 +109,9 @@ export default {
     decay() {
       return `${this.amount - this.holdAvailableAmount}`
     },
+    validLink() {
+      return new Date(this.validUntil) > new Date()
```

Comment:
This I am not sure of, but consider this:

```suggestion
      return (new Date(this.validUntil)).getTime() > (new Date()).getTime()
```

Reply:
no